cypress/page/products-list.page.ts

```diff
@@ -0,0 +1,19 @@
+class ProductListPage {
+    private addShirtCart: string;
+
+    private proceedToCart: string;
```

Comment:
could you add to the attributes of each page object the type of element we are interacting with? for example in this case, the element is a Button